backend/app/Http/Controllers/CategoryController.php

```diff
@@ -67,9 +84,13 @@ public function edit($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(Request $request, Category $category)
     {
-        //
+        $category->update($request->all());
```

Comment:
```suggestion
        $category->update($request->except('_token'));
```
change to `$request->except('_token')`